public_html/css/sass/game.scss

```diff
@@ -0,0 +1,76 @@
+$total-height: 100%;
+
+#game-view, #canvas-view, #color-palette-view, 
+.total-area {
+  height: $total-height;
+}
+
+#canvas-view, #color-palette-view {
+  background-color: white;
+  border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.